<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QDialog" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>548</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ATLAS Configuration</string>
  </property>
  <widget class="QTabWidget" name="settingsTabs">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>741</width>
     <height>471</height>
    </rect>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabGeneral">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QCheckBox" name="enableLabModeCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>211</width>
       <height>18</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>In Simulation Mode seismic and hydraulic events are played back from an existing catalog.</string>
     </property>
     <property name="text">
      <string>Run ATLAS in Lab Mode</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="openLastProjectOnStartup">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>191</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Open last project on startup</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSimulation">
    <attribute name="title">
     <string>Lab Mode</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>691</width>
       <height>91</height>
      </rect>
     </property>
     <property name="title">
      <string>Simulation Speed</string>
     </property>
     <widget class="QLabel" name="speedLabel">
      <property name="geometry">
       <rect>
        <x>214</x>
        <y>63</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="speedBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>215</x>
        <y>55</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string>x</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10000</number>
      </property>
      <property name="value">
       <number>1000</number>
      </property>
     </widget>
     <widget class="QRadioButton" name="simulateAFAPRadioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>161</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>As fast as possible</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="simuateAtSpeedFactorRadioButton">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>191</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>With a specific speed factor:</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabForecasting">
    <attribute name="title">
     <string>Forecasting</string>
    </attribute>
    <widget class="QGroupBox" name="schedulingBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>691</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Scheduling</string>
     </property>
     <widget class="QLabel" name="forecastIntervalLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>151</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Compute forecasts every</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="forecastIntervalBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>26</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string> h</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
      <property name="value">
       <number>6</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="rateIntervalBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>56</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string> min</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>60</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="rateIntervalLabel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Compute seismic rates every</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="forLabel">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>30</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>for</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="forecastBinNoBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>26</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string>x</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
      <property name="value">
       <number>6</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="forecastBinTimeBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>26</y>
        <width>71</width>
        <height>25</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="frame">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="suffix">
       <string> h</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
      <property name="value">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLabel" name="futureLabel">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>30</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>into the future</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="outputBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>260</y>
       <width>691</width>
       <height>100</height>
      </rect>
     </property>
     <property name="title">
      <string>Output</string>
     </property>
     <widget class="QCheckBox" name="writeResultsToFileCheckBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Write forecasting results to disk</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="outputPathLabel">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>65</y>
        <width>251</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>~/Desktop/AtlasResults</string>
      </property>
     </widget>
     <widget class="QPushButton" name="selectOutputDirButton">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Directory...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clearOutputDirButton">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>60</y>
        <width>161</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Output Directory</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="modelsBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>691</width>
       <height>121</height>
      </rect>
     </property>
     <property name="title">
      <string>Models</string>
     </property>
     <widget class="QCheckBox" name="enableRjCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>Reasenberg - Jones</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="entableEtasCheckBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>141</width>
        <height>18</height>
       </rect>
      </property>
      <property name="text">
       <string>ETAS</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="rjConfigButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>25</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Configure...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="etasConfigButton">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>55</y>
        <width>110</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Configure...</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>500</y>
     <width>110</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="resetToDefaultButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset to Default</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
